packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -452,4 +473,15 @@ $accent-blue: #117ac9;
 		box-shadow: inset 0 0 0 1px var( --wp-admin-theme-color-darker-10 );
 		color: var( --wp-admin-theme-color-darker-10 );
 	}
+
+	@media ( max-width: $break-mobile ) {
+		width: 100%;
+	}
+}
+
+.domain-picker__action {
```

Comment:
Personally I feel like we should be writing mobile-first CSS and use media queries and `min-width` to target tablet and above sizes rather than using desktop-first and using the `max-width` to target mobile/small devices. In which case the below CSS would become (untested):

```
.domain-picker__action {
	margin-top: rem( 16px );
	width: 100%;

	@include break-medium {
		margin-top: 0;
		width: auto;
	}
}
```

Reply:
I agree with the mobile-first approach. It complements the cascading nature of CSS, hence our code would be better comprehensible if we follow the convention. I'll update it and change it in other places as well :)